.github/workflows/test_and_deploy.yml

```diff
@@ -69,9 +70,27 @@ jobs:
     needs: build
     strategy:
       matrix:
-        scipy-version: [">=0.16,<1.4", ">=1.4,<=1.7.3"]
-        pyver: [3.6, 3.7, 3.8]
+        scipy-version: [">=1.4,<1.11.0"]
+        pyver: [3.8, 3.9, "3.10", 3.11]
         runs-on: [macos-latest, ubuntu-latest, windows-latest]
+        numpy-version: ["<1.22.0", ">=1.22.0"]
+        exclude:
+          - pyver: 3.11
+            numpy-version: "<1.22.0"
+        include:
```

Comment:
Why do we need to include these? I thought these would be covered by the martrix?

Reply:
There isn't a deprecation doc for scipy and numba-scipy as far as I'm aware. 
I added them as extra tests to the matrix because only 3.8 is supported by the older scipy versions (which in turn requires an older numpy version) and previously that configuration was tested.

Reply:
OK, I see, well, that's fine, I guess.